src/coreclr/interpreter/compiler.cpp

```diff
@@ -5512,7 +5519,10 @@ void InterpCompiler::GenerateCode(CORINFO_METHOD_INFO* methodInfo)
                 {
                     MergeStackTypeInfo(m_pStackBase, pNewBB->pStackState, pNewBB->stackHeight);
                     // This is relevant only for copying the vars associated with the values on the stack
-                    memcpy(m_pStackBase, pNewBB->pStackState, pNewBB->stackHeight * sizeof(StackInfo));
+                    if (pNewBB->stackHeight != 0)
```

Comment:
Same question as `ZeroMemory`/`memcpy`, the length should be fine. Is this because `m_pStackBase` or `pNewBB->pStackState` is actually `NULL`? If so, that should be the check, the length isn't the interesting part.

Reply:
Let's keep with the current behavior.